app/models/user.rb

```diff
@@ -1,6 +1,26 @@
 # frozen_string_literal: true
 
 class User < ApplicationRecord
+  PROFILE_IMAGE_EXTENSIONS = %w[image/jpg image/png image/gif].freeze
+  MAX_PROFILE_IMAGE_SIZE = 15.megabytes
+  RESUZE_IMAGE_SIZE = [800, 800].freeze
+
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+  has_one_attached :profile_image
+  after_save :resize_large_image
+  validates :profile_image, content_type: { in: PROFILE_IMAGE_EXTENSIONS, message: I18n.t('.activerecord.errors.messages.content_type_invalid') },
+                            size: { less_than: MAX_PROFILE_IMAGE_SIZE, message: I18n.t('.activerecord.errors.messages.size_invalid') }
```

Comment:
```suggestion
  validates :profile_image, content_type: { in: PROFILE_IMAGE_EXTENSIONS },
                            size: { less_than: MAX_PROFILE_IMAGE_SIZE }
```

メッセージは命名規則から自動的に取得するようにできませんかね？

Reply:
@JunichiIto 
手動で設定するのではなく、自動で出力されるんですね....
message属性を削除しました。